examples/material_gallery/lib/demo/list_demo.dart

```diff
@@ -0,0 +1,195 @@
+              )
+            ),
+            new ListItem(
+              isDense: true,
+              primary: new Text('Show Avatar'),
+              right: new Checkbox(
+                value: showAvatar,
+                onChanged: (bool value) {
+                  setState(() {
+                    showAvatar = value;
+                  });
+                }
+              )
+            ),
+            new ListItem(
+              isDense: true,
+              primary: new Text('Show Icon'),
+              right: new Checkbox(
+                value: showIcon,
+                onChanged: (bool value) {
+                  setState(() {
+                    showIcon = value;
+                  });
+                }
+              )
+            ),
+            new ListItem(
+              isDense: true,
+              primary: new Text('Dense Layout'),
+              right: new Checkbox(
+                value: isDense,
+                onChanged: (bool value) {
+                  setState(() {
+                    isDense = value;
+                  });
+                }
+              )
+            )
+          ]
+        )
+      );
+    });
+  }
+
+  Widget buildListItem(BuildContext context, String item) {
+    Widget secondary;
+    if (itemSize == ListDemoItemSize.twoLine) {
+      secondary = new Text(
+        "Additional item information."
+      );
+    } else if (itemSize == ListDemoItemSize.threeLine) {
+      secondary = new Text(
+        "Even more additional list item information appears on line three. "
+      );
+    }
+    return new ListItem(
+      isThreeLine: itemSize == ListDemoItemSize.threeLine,
+      isDense: isDense,
+      left: showAvatar ? new CircleAvatar(child: new Text(item)) : null,
+      primary: new Text('This item represents $item'),
+      secondary: secondary,
+      right: showIcon ? new Icon(icon: 'action/info', color: Theme.of(context).disabledColor) : null
+    );
+  }
+
+  Widget build(BuildContext context) {
+    final String layoutText = isDense ? " - Dense" : "";
+    String  itemSizeText;
+    switch(itemSize) {
+      case ListDemoItemSize.oneLine:
+        itemSizeText = 'Single-Line List';
+        break;
+      case ListDemoItemSize.twoLine:
+        itemSizeText = 'Two-Line List';
+        break;
+      case ListDemoItemSize.threeLine:
+        itemSizeText = 'Three-Line List';
+        break;
+    }
+    return new Scaffold(
+      key: scaffoldKey,
+      toolBar: new ToolBar(
+        center: new Text('${itemSizeText}${layoutText}'),
+        right: <Widget>[
+          new IconButton(
+            icon: "av/sort_by_alpha",
+            tooltip: 'Sort',
+            onPressed: () {
+              setState(() {
+                reverseSort = !reverseSort;
+                items.sort((String a, String b) => reverseSort ? a.compareTo(b) : b.compareTo(a));
+              });
+            }
+          ),
+          new IconButton(
+            icon: "navigation/more_vert",
```

Comment:
I don't think this icon should be used to show a bottom sheet. Maybe action/settings?